src/server_manager/web_app/app.ts

```diff
@@ -349,11 +348,11 @@ export class App {
       const doSession = this.createDigitalOceanSession(accessToken);
```

Comment:
Let's get rid of createDigitalOceanSession.
One of the goals of this PR is to not have to deal with the "session". Also, you have doAccount and this.digitalOceanAccount. We should have only one account.

Account should be able to provide email and status.